src/System.IO.FileSystem.sln

```diff
@@ -11,6 +11,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.IO.FileSystem", "Sys
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.IO.FileSystem.Tests", "System.IO.FileSystem\tests\System.IO.FileSystem.Tests.csproj", "{57E8F8D4-0766-4CC7-B3F9-B243B81DB6A5}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.IO.FileSystem.Watcher", "System.IO.FileSystem.Watcher\src\System.IO.FileSystem.Watcher.csproj", "{77E702D9-C6D8-4CE4-9941-D3056C3CCBED}"
```

Comment:
We don't want to give Watcher its own .sln file?  DriveInfo has its own. (Primitives doesn't, but it's just a few enums.)